content/en/docs/concepts/configuration/manage-compute-resources-container.md

```diff
@@ -10,23 +10,45 @@ feature:
 
 {{% capture overview %}}
 
-When you specify a [Pod](/docs/concepts/workloads/pods/pod/), you can optionally specify how
-much CPU and memory (RAM) each Container needs. When Containers have resource
-requests specified, the scheduler can make better decisions about which nodes to
-place Pods on. And when Containers have their limits specified, contention for
-resources on a node can be handled in a specified manner. For more details about
-the difference between requests and limits, see
-[Resource QoS](https://git.k8s.io/community/contributors/design-proposals/node/resource-qos.md).
+When you specify a {{< glossary_tooltip term_id="pod" >}}, you can optionally specify how
+much resources each {{< glossary_tooltip text="Container" term_id="container" >}} needs.
+The most common resources you specify are CPU and memory (RAM), and there are others you might
+use as well.
+
+When you specify the resource _request_ for Containers, the scheduler can make better decisions
+about which nodes to place Pods on. And when you specify _limits_ for Containers, that lets
+the kubelet on a node manage contention for those resources.
+
```

Comment:
This paragraph confuses me a little, but maybe try:

> When you specify the resources a Container needs, the scheduler uses this information to decide which node to place a Pod on. When you specify the limits(? what are limits) for a container, a node's kublet manages contention for those resources.

Also, the idea of "resource contention" is confusing to me in general. It's mentioned all over in the docs so it's probably fine but... :|

Reply:
I'm not sure I can come up with wording that is clearer - sorry! I'm hoping that these changes are at least an improvement on what's there now.

Reply:
I've had a go!